src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const declarations = sourceString
```

Comment:
The variable name 'declarations' could be more descriptive. Consider renaming it to something that indicates it contains CSS declarations.

Reply:
done